### PR TITLE
Add support for proxy adapter

### PR DESCRIPTION
**Pull request by JasonKeirstead**
Fix JSON indenting

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -130,14 +132,95 @@ def __main__():
         help='Query String'
     )
 
+    host_parser = parent_subparsers.add_parser(HOST, help='Host a local query service, for testing and developmenr')
```
Comment (delliott90) [inline main.py]: Typo in the help message. I think you meant `development`.

Comment (JasonKeirstead) [reply main.py]: Thanks, fixed this one

File: stix_shifter/stix_transmission/src/modules/proxy/proxy_connector.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        self.delete_connector = self
+        self.query_connector = self
+        self.ping_connector = self
+
+    def ping(self):
+        response = requests.post("http://" + self.connection["proxy_host"] + ":" + self.connection["proxy_port"] + "/ping",
+            data=json.dumps({"connection":self.connection,"configuration":self.configuration}))
+        return response.text
+
+    def ping(self):
```
Comment (delliott90) [inline stix_shifter/stix_transmission/src/modules/proxy/proxy_connector.py]: Duplicate ping methods.

**Comment by delliott90** [comment]
Should the developer guide (linked from the readme) have some instructions on how to use the proxy adapter?

File: stix_shifter/stix_transmission/src/modules/proxy/proxy_connector.py (hunk 2 above)
Comment (JasonKeirstead) [reply stix_shifter/stix_transmission/src/modules/proxy/proxy_connector.py]: You're reviewing an old commit; see latest PR

Comment (delliott90) [reply stix_shifter/stix_transmission/src/modules/proxy/proxy_connector.py]: I am looking at the latest.

**Pull request merged**